addOns/postman/src/main/javahelp/org/zaproxy/addon/postman/resources/help/contents/postman.html

```diff
@@ -14,7 +14,7 @@ <H2>UI</H2>
 <ul>
 <li>Import a Postman Collection</li>
 </ul>
-The dialog allows providing a comma-separated list of variables as key-value pairs in the format <code>key1=value1,key2=value2,...</code>
+Any variables defined in the collection will be replace with their values. Additionally, the dialog allows providing a comma-separated list of variables as key-value pairs in the format <code>key1=value1,key2=value2,...</code>
```

Comment:
Shall we still keep variables through the dialog? It can be useful as they have precedence over the collection ones.

Reply:
I think so.